lib/internal/modules/cjs/loader.js

```diff
@@ -1173,10 +1192,10 @@ function wrapSafe(filename, content, cjsModuleInstance) {
 Module.prototype._compile = function(content, filename) {
   let moduleURL;
   let redirects;
-  if (policy?.manifest) {
+  if (policy()?.manifest) {
```

Comment:
nit: it might be worth caching the value here to save a few `policy()` calls below.
```suggestion
  const manifest = policy()?.manifest
  if (manifest != null) {
```